service-workers/service-worker/resources/import-mime-type-worker.py

```diff
@@ -1,10 +1,12 @@
+from wptserve.utils import isomorphic_decode
+
 def main(request, response):
-    if 'mime' in request.GET:
+    if b'mime' in request.GET:
         return (
-            [('Content-Type', 'application/javascript')],
-            "importScripts('./mime-type-worker.py?mime={0}');".format(request.GET['mime'])
+            [(b'Content-Type', b'application/javascript')],
+            u"importScripts('./mime-type-worker.py?mime={0}');".format(isomorphic_decode(request.GET[b'mime']))
```

Comment:
```suggestion
            b"importScripts('./mime-type-worker.py?mime={0}');".format(request.GET[b'mime'])
```

Reply:
This might cause the following error in Python 3 run:
AttributeError: 'bytes' object has no attribute 'format'

Reply:
Oh, sorry, I tested b''.format() in Python 2 only and thought it would be OK. Not so :)

How about using `%` instead?

Reply:
Or just `+` would work.

Reply:
Updated. Thank you!